bepis.json

```diff
@@ -89,6 +89,33 @@
         "bergus",
         "hergus",
         "beps!",
+        "sprungy",
+        "du#m#b",
```

Comment:
mmk!! will do - didn't realise that @ the time i was falling asleep and trying to think of words that ended in "mb" because stuff like "jimb" and "crumbis" makes me lose it